### DISCUSSION TITLE
Sparkle Always shows new update is available

### DISCUSSION THREAD

**Discussion by Beingpax** [sparkle-project/Sparkle#2631] [Q&A]
### Description of the problem
It always shows new update is available even after the app is updated to latest version. 
![CleanShot 2024-09-16 at 21 47 53](https://github.com/user-attachments/assets/b17d1474-3059-464a-ae0f-21e8b7273d52)

This is the appcast.xml file. 
`<?xml version="1.0" standalone="yes"?>
<rss xmlns:sparkle="http://www.andymatuschak.org/xml-namespaces/sparkle" version="2.0">
    <channel>
        <title>ZenFocus</title>
        <item>
            <title>0.1.14</title>
            <pubDate>Mon, 16 Sep 2024 20:03:23 +0545</pubDate>
            <sparkle:version>114</sparkle:version>
            <sparkle:shortVersionString>0.1.14</sparkle:shortVersionString>
            <sparkle:minimumSystemVersion>14.0</sparkle:minimumSystemVersion>
            <enclosure url="https://github.com/Beingpax/ZenFocus-App/releases/download/v0.1.14/ZenFocus.dmg" length="3335397" type="application/octet-stream" sparkle:edSignature="YznFca7dhV8asiaPNT3zYCeDy5QxepA7nmqIpXIKMYP9q61DBB07RgPWr/FvcgCfOhGGFWaANzcAUnVgCbevBQ=="/>
        </item>
    </channel>
</rss>`

When I use the sparkle:version as 0.1.14, it always shows this error. 
![CleanShot 2024-09-16 at 21 51 45](https://github.com/user-attachments/assets/c26d05ba-cbea-4f7f-a15d-562651fd2f97)

### Do you use Sandboxing in your app?
No

### Version of `Sparkle.framework` in the latest version of your app
2.6.4

### Version of `Sparkle.framework` in the old version of app that your users have (or N/A)
2.6.4

**Comment by zorgiepoo** [comment]
Sparkle thought your app has a CFBundleVersion of 1 in your app's Info.plist, instead of 114. The CFBundleVersion is what is compared with sparkle:version

**Reply by zorgiepoo** [reply]
The update check seems to work fine on my end with your build https://github.com/Beingpax/ZenFocus-App/releases/download/v0.1.14/ZenFocus.dmg -- it's up to date.
